arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/SloadSection.java

```diff
@@ -43,23 +46,23 @@ public class SloadSection extends TraceSection implements PostRollbackDefer {
   final EWord valueCurrent;
   final short exceptions;
 
-  public SloadSection(Hub hub, WorldView world) {
+  public SloadSection(Hub hub, WorldView worldView) {
     // exceptional case:   1 (stack row) + 5 (non stack rows)
     // unexceptional case: 1 (stack row) + 4 (non stack rows)
     super(
         hub,
         (short)
             (hub.opCode().numberOfStackRows() + (Exceptions.any(hub.pch().exceptions()) ? 5 : 4)));
 
-    this.world = world;
-    this.address = hub.messageFrame().getRecipientAddress();
-    this.deploymentNumber = hub.currentFrame().accountDeploymentNumber();
-    this.storageKey = Bytes32.leftPad(hub.messageFrame().getStackItem(0));
-    this.incomingWarmth = hub.messageFrame().getWarmedUpStorage().contains(address, storageKey);
-    this.valueOriginal =
-        EWord.of(world.get(address).getOriginalStorageValue(UInt256.fromBytes(storageKey)));
-    this.valueCurrent = EWord.of(world.get(address).getStorageValue(UInt256.fromBytes(storageKey)));
-    this.exceptions = hub.pch().exceptions();
+    world = worldView;
+    address = hub.messageFrame().getRecipientAddress();
+    deploymentNumber = hub.deploymentNumberOf(address);
+    storageKey = Bytes32.leftPad(hub.messageFrame().getStackItem(0));
+    incomingWarmth = hub.messageFrame().getWarmedUpStorage().contains(address, storageKey);
+    valueOriginal =
+        EWord.of(worldView.get(address).getOriginalStorageValue(UInt256.fromBytes(storageKey)));
+    valueCurrent = EWord.of(worldView.get(address).getStorageValue(UInt256.fromBytes(storageKey)));
+    exceptions = hub.pch().exceptions();
```

Comment:
The disparity between the underlying addresses (accountAddress vs bytecodeAddress) seems to have caused the above.